plugins/microservices/msi_tests/libmsi_test_scoped_client_identity.cpp

```diff
@@ -0,0 +1,146 @@
+/// \file
+
+#include "getRodsEnv.h"
+#include "irods_exception.hpp"
+#include "irods_ms_plugin.hpp"
+#include "irods_re_structs.hpp"
+#include "msParam.h"
+#include "msi_assertion_macros.hpp"
+#include "scoped_client_identity.hpp"
+
+#include <string>
+#include <exception>
+
+namespace
+{
+    auto test_basic_usage() -> int
+    {
+        IRODS_MSI_TEST_BEGIN("scoped_client_identity - basic usage")
+
+        const std::string username = "rods";
+        const std::string zone = "tempZone";
```

Comment:
These could be derived from the client environment like in `test_updates_zone`, right?

Reply:
They can. I chose not to because the test is just about showing how the class handles strings. The strings could have been anything at this point lol.